Sources/PrimerSDK/Classes/User Interface/Confirm Mandate/ConfirmMandateViewModel.swift

```diff
@@ -106,8 +106,10 @@ class ConfirmMandateViewModel: ConfirmMandateViewModelProtocol {
                     completion(error)
                 case .success(let token):
                     state.directDebitMandate = DirectDebitMandate(address: Address())
-                    let settings: PrimerSettingsProtocol = DependencyContainer.resolve()
-                    settings.onTokenizeSuccess(token, completion)
+                    // FIXME: Please review this carefully. I believe that the authorizePayment delegate method should not be called since Direct Debit can only be vaulted. 
```

Comment:
True, a direct debit token cannot be used to auth/capture a payment. However, it might still make sense to pass it back to the merchant as they might want to store the token or something? Wdyt?

Reply:
Makes sense, fixed!